code/controllers/subsystem/ticker.dm

```diff
@@ -424,7 +424,7 @@ SUBSYSTEM_DEF(ticker)
 
 	for(var/mob/living/carbon/human/player in GLOB.human_mob_list)
 		if(player.mind)
-			if(player.job == "Commanding Officers")
+			if(player.job == "Commanding Officer")
```

Comment:
```suggestion
			if(player.job == JOB_CO)
```
if only someone had used the defines in the first place it would not have happened

Reply:
(also fix changelog)